llvm/unittests/ProfileData/MemProfTest.cpp

```diff
@@ -265,7 +265,7 @@ TEST(MemProf, PortableWrapper) {
   EXPECT_EQ(3UL, ReadBlock.getAllocCpuId());
 }
 
-TEST(MemProf, RecordSerializationRoundTrip) {
+TEST(MemProf, RecordSerializationRoundTripVersion0) {
```

Comment:
Should we also have a test for version 1?

Reply:
Maybe I'll rename the test to `Version0AndVerion1` or something because the serialization format is the same between those two versions.

Reply:
Oh I overlooked the fact that the serialization format of these records is not changed between versions 0 and 1. So another option is to simply add a comment here that version 1 is the same.

Reply:
Oops, I read your comment after I updated the patch, but, yes, I did put a comment there.